.env.baklava

```diff
@@ -31,13 +31,13 @@ CELOSTATS_BANNED_ADDRESSES=""
 CELOSTATS_RESERVED_ADDRESSES=""
 
 GETH_NODE_DOCKER_IMAGE_REPOSITORY="us.gcr.io/celo-org/geth"
-GETH_NODE_DOCKER_IMAGE_TAG="1.2.2"
+GETH_NODE_DOCKER_IMAGE_TAG="1.3.0-beta"
 
 GETH_BOOTNODE_DOCKER_IMAGE_REPOSITORY="us.gcr.io/celo-testnet/geth-all"
-GETH_BOOTNODE_DOCKER_IMAGE_TAG="master"
+GETH_BOOTNODE_DOCKER_IMAGE_TAG="2dfdf20bf76d8e50e22106eb816a96311de8123d"
```

Comment:
I believe this can also be 1.3.0-beta, that tag points to the same as this hash.